server/BackendModule/src/main/java/io/egen/entity/ShowDetails.java

```diff
@@ -0,0 +1,246 @@
+package io.egen.entity;
+
+
+import java.util.Date;
+import java.util.List;
+
+
+
+
+
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.GenericGenerator;
+
+
+
+
+
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name="ShowDetails")
+public class ShowDetails {
+	@Id
+    @GenericGenerator(strategy="uuid2", name="myuuid")
+    @GeneratedValue(generator="myuuid")
+    private String MovieId;
+    private String Title;
+    private int Year;
+    private String Rated;
+    
+    private Date Released;
+    private String Runtime;
+    private String Genre;
+    private String Director;
+    private String Writer;  
+	private String Actors ;
+	 @Column(length = 400)
+    private String Plot;
+	private String Language;
+	private String Country;
+    private String Awards ;
+    private String Poster;
+    private String Metascore; 
+    private double imdbRating ;
+  
+    private long imdbVotes;
+    private String imdbID;
+    private String Type;
+    
+	public String getTitle() {
```

Comment:
You are already using lombok. Hence, use `@Data` and avoid writing these getter setters